app/src/routes/project/EditPanel/ManualTab.tsx

```diff
@@ -85,21 +96,21 @@ const ManualTab = observer(() => {
             return (
                 <NestedInputs
                     elementStyles={elementStyles}
-                    updateElementStyle={updateElementStyle}
+                    updateElementStyle={updateElementStyle(elementStyles[0])} // TODO: this needs a more general approach
```

Comment:
Out of scope for this PR but I don't like the updateElementStyle being passed around pattern very much which was done for historical reasons. I will very likely move this into the EditorEngine in a future refactor. 

Same with the way elementStyles are handled and generated. I think it's too complicated. 

Nothing actionable here, just FYI.